build.gradle.kts

```diff
@@ -24,22 +24,24 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-
 import Build_gradle.Module
-import com.google.protobuf.gradle.protobuf
+import io.spine.internal.dependency.ErrorProne
 import io.spine.internal.dependency.Protobuf
 import io.spine.internal.dependency.Spine
-import io.spine.internal.dependency.ErrorProne
 import io.spine.internal.gradle.javac.configureErrorProne
 import io.spine.internal.gradle.javac.configureJavac
 import io.spine.internal.gradle.kotlin.applyJvmToolchain
 import io.spine.internal.gradle.kotlin.setFreeCompilerArgs
 import io.spine.internal.gradle.standardToSpineSdk
-import org.gradle.api.Project
 import org.gradle.api.tasks.testing.logging.TestLogEvent
-import org.gradle.kotlin.dsl.invoke
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
+object BuildSettings {
```

Comment:
In Gradle, `buildscript` section is a special one. It tells how to perform the build itself in terms of classpath etc. One should think of it as of a configuration for Gradle, so to speak. It is also isolated from the rest of the build in classpath, imports and variables.

In `plugins` sections our allowed actions are also limited. For instance, AFAIK, we cannot use external variables inside of it.

Therefore I suggest to move `BuildSettings` to go even after `plugins`, right before the actual code which we write to describe the build stages.

Reply:
Done.